src/plugins/dataflow-tool/nodes/factories/math-rete-node-factory.tsx

```diff
@@ -46,8 +46,8 @@ export class MathReteNodeFactory extends DataflowReteNodeFactory {
       if (isNaN(n1) || isNaN(n2)) {
         result = NaN;
       } else {
-        // NaNs are for propogating lack of values. Actual math erros like
-        // divide-by-zero should output 0.
+        // NaNs are for propagating lack of values.
+        // Actual math errors like divide-by-zero should output 0.
         result = nodeOperationTypes.method(n1, n2) || 0;
```

Comment:
I'm not sure I agree with this approach, but since you're only fixing typos in the comments (which I wholly support) any discussion of its merits are beyond the scope of this PR.